### DISCUSSION TITLE
Shortcut to add image to node

### DISCUSSION THREAD

**Discussion by Bender11011** [phase1geo/Minder#636] [Ideas]
I quite enjoy using Minder however there is one small fix that would help a lot. It would be great if a keyboard shortcut to add an image to a node was put in a future update, as its the only change node option without one.

**Comment by phase1geo** [comment]
I have gone ahead and added the Shift-I keyboard shortcut for this feature request.  The change is available in the master branch of Minder and will be available in the upcoming 1.17.0 release.  Thank you for making this request!

**Comment by Bender11011** [comment]
Just tried it out, thanks!
